### ISSUE TITLE
About Dream Pre-train

### ISSUE THREAD

**Issue by sglucas** [ZHZisZZ/dllm-trainer#78]
Hi, I would like to follow your provided command about Dream pre-traning, and I found the training loss and gradient norm is pretty large, (such as loss is 121). May I ask whether I still need to tune the hyper-parameters and the authors try to pre-train dream, evaluate its performance. 

Best regards
